e2e/element.test.js

```diff
@@ -120,6 +120,66 @@ describe('elements', () => {
         const link = await browser.findElement('partial link text', 'new tab')
         expect(await browser.getElementText(link[ELEMENT_KEY])).toBe('open new tab')
     })
+
+    it('should be able to do a drag&drop', async () => {
+        await browser.performActions([{
+            type: 'pointer',
+            id: 'finger1',
+            parameters: {
+                pointerType: 'mouse'
+            },
+            actions: [{
+                type: 'pointerMove',
+                duration: 0,
+                x: 65,
+                y: 544
+            }, {
+                type: 'pointerDown',
+                button: 0
+            }, {
+                type: 'pause',
+                duration: 10
+            }, {
+                type: 'pointerMove',
+                duration: 100,
+                origin: 'pointer',
+                x: 1,
+                y: -251
+            }, {
+                type: 'pointerUp',
+                button: 0
+            }]
+        }])
+
+        const elem = await browser.findElement('css selector', '.searchinput')
+        expect(await browser.getElementProperty(elem[ELEMENT_KEY], 'value'))
+            .toBe('Dropped!')
+    })
+
+    it('should be able to use keys command', async () => {
```

Comment:
Mind if we also test keys like SHIFT and an emoji?

Reply:
I added a test that includes an emoji. It seems that using shift to get characters written as uppercase seems to be not possible. I would maybe create a bug to investigate this later but wouldn't see it as a show stopper.